src/liflines/selectdb.c

```diff
@@ -181,7 +182,8 @@ open_or_create_database (INT alteration, STRING *dbused)
 	strupdate(dbused, dbdir);
 
 	/* Is user willing to make a new db ? */
-	if (!ask_yes_or_no_msg(_(qSnodbse), _(qScrdbse))) 
+        snprintf(newmsg,sizeof(newmsg),"%s '%s' ?",qScrdbse,*dbused);
```

Comment:
Then this becomes:

```
snprintf(newmsg,sizeof(newmsg),qScrdbse,*dbused);
```

Doing it this way ensures that when the message is translated, the %s can get put into the right spot.  Also knowing that the message is a question is important, so the question mark should be in the message, not the snprintf here.